tests/integration/templates/etcd/etcd.service

```diff
@@ -11,6 +11,7 @@
     LimitNOFILE=40000
     TimeoutStartSec=0
 
+    Environment=ETCD_UNSUPPORTED_ARCH=$ARCH
```

Comment:
this is required to run on arm64
https://etcd.io/docs/v3.6/op-guide/supported-platform/

Reply:
let's just bump the version. We don't have any dependencies/reasons to run on v3.3